src/plugins/interactive-media/interactive-media.less

```diff
@@ -115,18 +113,71 @@
   }
 
   &.is-active {
-    color: @edu-green-icon-color;
+    color: @edu-success-color;
   }
 }
 
 .InteractiveMediaDisplay-overlay {
   position: absolute;
+  padding: 15px;
```

Comment:
Or maybe the equivalent `@edu-content-padding`?

Reply:
Changed!